ppr-ui/src/views/mhrInformation/MhrInformation.vue

```diff
@@ -124,6 +132,9 @@ export default defineComponent({
       setShowGroups
     } = useHomeOwners()
 
+    const options = unsavedChangesDialog
+    var showCancelDialog = false
```

Comment:
Should probably use const.
That being said, doesn't this belong in the localState?

Reply:
Now initialized in localState..